### DISCUSSION TITLE
블로그 글 내부 이미지 관련 문의

### DISCUSSION THREAD

**Discussion by Sooho-Kim** [pronist/hELLO#84] [Q&A] [answered]
안녕하세요! 스킨을 이쁘게 잘 만들어주셔서 먼저 감사하다는 말씀을 드립니다!
버그 리포트인지는 정확하게 모르겠으나, 혹시 동일 증상이 나왔는지 궁금하네요!

제 경우에는 이미지가 작게 노출되는 현상이 있습니다.
![화면 캡처 2021-05-02 133952](https://user-images.githubusercontent.com/67626878/116802203-0e41f580-ab4c-11eb-94e9-4ebcbd87b634.png)

아래는 원래 사진 형태입니다. 새로고침 누르면 다시 원래 사진으로 돌아오긴 합니다.
![화면 캡처 2021-05-02 134018](https://user-images.githubusercontent.com/67626878/116802213-2285f280-ab4c-11eb-829c-1215bb33f913.png)

티스토리 자체적인 문제일까요? 확인을 부탁드립니다! 감사합니다.

**Comment by pronist** [comment, marked as answer]
@Sooho-Kim 안녕하세요. 스킨 내부에서는 이미지와 관련해서 크기를 리사이징하는 것은 없고 중앙으로 정렬하는 것은 있습니다. 그 외에 너무 큰 이미지에 대해 **1100px** 으로 제한하는 것은 있으나 작은 이미지는 특별히 건드리지는 않습니다 :) 이 문제는 처음 보고받는 것이어서 티스토리의 문제일 가능성이 큽니다.

**Comment by Sooho-Kim** [comment]
답변 감사합니다 !!
